spec/factories/involved_civilian.rb

```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
-  factory :involved_civilian, class: InvolvedCivilian do
```

Comment:
For my information -- what's the purpose of `class` in FactoryGirl, and why are we removing it?